src/Assistant.py

```diff
@@ -1,16 +1,19 @@
 import re
 from datetime import datetime
-
+import threading
 import Support
 from VoiceInterface import VoiceInterface
 
 LISTENING_ERROR = "Say that again please..."
 
+
```

Comment:
two line gap is enough...no need for having three blank lines

Reply:
Kept only two blank lines.